compiler/qsc/src/interpret/stateful.rs

```diff
@@ -358,13 +357,23 @@ impl Interpreter {
     ) -> Result<Value, Vec<LineError>> {
         let mut result = Value::unit();
 
-        let mut fragments = self.compiler.compile_fragments(line).map_err(|errors| {
-            let source = line.into();
-            errors
-                .into_iter()
-                .map(|error| LineError(WithSource::new(Arc::clone(&source), error.into(), None)))
-                .collect::<Vec<_>>()
-        })?;
+        let line_label = format!("line_{}", self.lines);
+        self.lines += 1;
```

Comment:
nitpick: it would be nice if this were wrapped in a function that returned the string and performed the increment all together, something like `self.next_line_label()`. Even though this is only done one place, it makes clear that the increment and construction of a label should be done together. Not a high priority though.

Reply:
I like this a lot, thanks!